src/components/wallet/methods/updateAccount.ts

```diff
@@ -8,18 +8,34 @@ import { handleError } from '@services/error'
 import { Wallet } from '../wallet'
 
 export default async function updateAccount(this: Wallet) {
+  function cbQuery({ this: wallet }) {
+    wallet.server
+      .claimableBalances()
+      .claimant(wallet.account.publicKey)
+      .call()
+      .then(function (resp) {
+        resp.records.forEach((record) => {
+          wallet.logger.response(
+            'Claimable Balances Available ',
+            loOmit(record, ['_links', 'paging_token', 'self'])
+          )
+        })
+        console.log(resp)
+      })
+      .catch(function (err) {
+        wallet.error = handleError(err)
+        console.error(err)
```

Comment:
I would image `handleError()` pushes the message to the relevant channels right? `console.error()` probably isn't needed.